runtime/src/accounts.rs

```diff
@@ -943,7 +934,7 @@ mod tests {
         ka: &Vec<(Pubkey, Account)>,
         fee_calculator: &FeeCalculator,
         error_counters: &mut ErrorCounters,
-    ) -> Vec<Result<(InstructionAccounts, InstructionLoaders)>> {
+    ) -> Vec<Result<(Vec<Account>, Vec<Vec<(Pubkey, Account)>>)>> {
```

Comment:
Why not update the type aliases? Otherwise, if this was by chance the last references to them, delete them?